adafruit_magtag/network.py

```diff
@@ -50,10 +56,10 @@ class Network(NetworkBase):
     def __init__(
         self,
         *,
-        status_neopixel=None,
-        extract_values=True,
-        debug=False,
-    ):
+        status_neopixel: Optional[Union[microcontroller.Pin, neopixel.NeoPixel]] = None,
```

Comment:
This is correct for status_neopixel. I left out NeoPixel from my other comment accidentally, but the other status_neopixel can be copied from this one here.